### PR TITLE
Refactor setting annotations and labels on backed-pvc

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

### CODE REVIEW

**Comment by ashish-amarnath** [comment]
I had to do a rebase to open this PR. I'll be taking this PR through a round of testing. Holding till then.

**Comment by ashish-amarnath** [comment]
This looked good on testing. Ready for 👀

File: velero-csi-plugin/csi_snapshotter_test.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+				},
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		before := tc.pvc.DeepCopy()
+		setPVCAnnotationsAndLabels(&tc.pvc, tc.snapshotName, tc.backupName)
+
+		if tc.pvc.Annotations == nil {
```
Comment (nrb) [inline velero-csi-plugin/csi_snapshotter_test.go]: For conditions like this, we can use `"github.com/stretchr/testify/assert"`. It has helpers like `assert.NotNil` and `assert.True`. IMO, that dependency is worthwhile.

Comment (ashish-amarnath) [reply velero-csi-plugin/csi_snapshotter_test.go]: Yes. I have another PR in the pipeline to switch to `stretchr` for all tests.

Comment (nrb) [reply velero-csi-plugin/csi_snapshotter_test.go]: Ah, ok, then I'll remove the request for changes and we can get them all in one.

**Pull request merged**